lib/processEventDataHelper.js

```diff
@@ -1,105 +1,100 @@
-/*eslint no-console: 0 */
-'use strict';
+/* eslint no-console: 0 */
+
 const moment = require('moment-timezone');
 const _ = require('lodash');
 const Q = require('q');
 
 function checkRequiredFields(cfg, messageBody) {
-    console.log('Checking required fields ...');
-    if (!messageBody.start) {
-        throw new Error('Start Time missing! This field is required!');
-    }
-    if (!messageBody.end) {
-        throw new Error('End Time missing! This field is required!');
-    }
-    if (!cfg.timeZone) {
-        throw new Error('Time Zone missing! This field is required!');
-    }
-    if (!cfg.calendarId) {
-        throw new Error('Calendar ID missing! This field is required!');
-    }
-    if ((cfg.bodyContentType) && (!messageBody.body)) {
-        throw new Error('Body Type provided, but Body Content is missing!');
-    }
+  console.log('Checking required fields ...');
+  if (!messageBody.start) {
+    throw new Error('Start Time missing! This field is required!');
+  }
+  if (!messageBody.end) {
+    throw new Error('End Time missing! This field is required!');
+  }
+  if (!cfg.timeZone) {
+    throw new Error('Time Zone missing! This field is required!');
+  }
+  if (!cfg.calendarId) {
+    throw new Error('Calendar ID missing! This field is required!');
+  }
+  if ((cfg.bodyContentType) && (!messageBody.body)) {
+    throw new Error('Body Type provided, but Body Content is missing!');
+  }
 }
 
 function formatDate(date, timeZone, format) {
-
-    //Valid date string
-    if (moment(date).isValid()) {
-    //Valid ISO_8601 string
-        if ((moment(date, moment.ISO_8601).isValid())) {
-            console.log(`Date ${date} has a known ISO-8601 format.`);
-            return moment.tz(date, timeZone).format(format);
-        }
-        // non ISO_8601 string
-        throw new Error(`non ISO-8601 date formats are currently not supported: ${date}.`);
+  // Valid date string
+  if (moment(date).isValid()) {
+    // Valid ISO_8601 string
+    if ((moment(date, moment.ISO_8601).isValid())) {
+      console.log(`Date ${date} has a known ISO-8601 format.`);
```

Comment:
Should we replace this `console.log` with `this.logger`?

Reply:
done